x/genutil/client/cli/init.go

```diff
@@ -31,6 +32,9 @@ const (
 
 	// FlagSeed defines a flag to initialize the private validator key from a specific seed.
 	FlagRecover = "recover"
+
+	// FlagStakingBondDenom defines a flag to specify the staking token in the genesis file.
+	FlagStakingBondDenom = "stakingBondDenom"
```

Comment:
flags are dash-cased.

```suggestion
	FlagStakingBondDenom = "staking-bond-denom"
```